.gitattributes

```diff
@@ -1,12 +1,15 @@
 /tests export-ignore
 /docs export-ignore
-.gitattributes export-ignore
-.gitignore export-ignore
-.appveyor.yml export-ignore
-.travis.yml export-ignore
-build.properties export-ignore
-build.xml export-ignore
-phpunit.xml.dist export-ignore
-run-all.sh export-ignore
+/.gitattributes export-ignore
+/.gitignore export-ignore
+/.appveyor.yml export-ignore
+/.travis.yml export-ignore
+/.scrutinizer.yml export-ignore
+/build.properties export-ignore
+/build.xml export-ignore
```

Comment:
Are these Ant build files still relevant?

Reply:
Same question crossed my mind. I guess not, we should remove them in another PR.